sql/interbase/init_relations.sql

```diff
@@ -198,7 +198,8 @@ CREATE TABLE problems (
     run_method         SMALLINT DEFAULT 0,
     players_count      VARCHAR(200),
     statement_url      VARCHAR(200) DEFAULT '',
-    explanation_url    VARCHAR(200) DEFAULT ''
+    explanation_url    VARCHAR(200) DEFAULT '',
+    sub_path           VARCHAR(200) DEFAULT '' NOT NULL
```

Comment:
1) Remove this commit from PR, it belongs to another
2) Remove NOT NULL, this column should be nullable
3) Rename to repo_path